### PR TITLE
Upgrade to fxtest-jenkins-pipeline library v1.10

### PR DESCRIPTION
**Pull request by stephendonner**
Passing ad-hoc job here: https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/stubattribution.adhoc/130/console

@davehunt r?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 __pycache__
 .cache
+.git
```
Comment (davehunt) [inline .gitignore]: I don't think you need to tell git to ignore itself. Did you mean to put this in `.dockerignore`?

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 FROM python:2.7-alpine
-RUN apk add --update git openssl
+RUN apk add --update openssl
```
Comment (davehunt) [inline Dockerfile]: If we move the `ircNotification` to run outside of the Docker image (like we did for Mozillians) then we can probably remove `openssl` too. Perhaps a follow-up?

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,6 @@
...
 *.pyc
 *.tox
```
Comment (davehunt) [inline .gitignore]: Can we remove this now?

File: .gitignore (hunk 1 above)
Comment (stephendonner) [reply .gitignore]: Er, brain fart; yeah.  Will fix.

File: Dockerfile (hunk 2 above)
Comment (stephendonner) [reply Dockerfile]: Oh, yes - forgot about that change; I can and will do that in a follow-up PR, sure.

**Comment by stephendonner** [comment]
@davehunt final r?

File: Dockerfile (hunk 2 above)
Comment (stephendonner) [reply Dockerfile]: Put up a PR, here: https://github.com/mozilla/stubattribution-tests/pull/100/files

**Pull request merged**